src/index.ts

```diff
@@ -11,6 +12,8 @@ function microserviceExists(microservice: "api" | "admin" | "site") {
     return fs.existsSync(`${microservice}/package.json`);
 }
 
+const isLocalDevelopment = !process.argv[1].includes("/_npx");
```

Comment:
This was the easiest way I found to check if the script is executed locally during development or via npx. 

Maybe we can think of something smarter.

Reply:
Why doesn't `process.env.NODE_ENV === "development"` work? 🤔

Reply:
NODE_ENV must be set explicitly. At this point it's `undefined` which is the default value.

Reply:
I found another way to do it by checking `npm_execpath` for `npx`. This seems less hacky to me. I also added the warning: "Not running via npx -> assuming local development. Scripts will run twice to ensure idempotency."

34c957f87f24d721602a1878ed6093e53ed5528e